src/main/java/com/ufcg/es5/BackendComplexoEsportivoUFCG/application/court/service/CourtServiceImpl.java

```diff
@@ -53,6 +53,15 @@ public CourtResponseDto update(CourtUpdateDto data, Long id) throws SaceResource
         return objectMapper.convertValue(court, CourtResponseDto.class);
     }
 
+    @Override
+    @Transactional
+    public void delete(Long id) throws SaceResourceNotFoundException {
+        Court court = repository.findById(id).orElseThrow(() -> new SaceResourceNotFoundException(
+                CourtExceptionMessages.COURT_WITH_ID_NOT_FOUND.formatted(id)
+        ));
+        repository.delete(court);
```

Comment:
usar o service, this.deleteById(id). Recomendo revisar os  métodos que são implementados pelo BasicService